### PR TITLE
Add OpenSCAD 2016.10.28

### PR DESCRIPTION
**Pull request by neutric**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-versions/pulls
[closed issues]: https://github.com/caskroom/homebrew-versions/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/openscad-dev.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'openscad-dev' do
```
Comment (vitorgalvao) [inline Casks/openscad-dev.rb]: Call it `snapshot` instead.

File: Casks/openscad-dev.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  version '2016.10.28'
+  sha256 '2ab89b438fc25bbfc7ebc1bd1aa131f2be3d7b4a70659d1fc5ef67f2c2c618c5'
+
+  url "http://files.openscad.org/snapshots/OpenSCAD-#{version}.dmg"
+  name 'OpenSCAD'
+  homepage 'http://www.openscad.org/'
```
Comment (vitorgalvao) [inline Casks/openscad-dev.rb]: Isn’t there a page specific to the snapshots?

Comment (neutric) [inline Casks/openscad-dev.rb]: ...and done. :)

**Comment by vitorgalvao** [comment]
Thank you.

**Comment by neutric** [comment]
Thanks for merging, Vítor!

**Pull request merged**